docbook/Developers_Guide/en-US/Plugins_Building_Source.xml

```diff
@@ -37,9 +37,10 @@ class ExamplePlugin extends MantisPlugin {
         $this->page = '';           # Default plugin page
 
         $this->version = '1.0';     # Plugin version string
-        $this->requires = array(    # Plugin dependencies, array of basename => version pairs
-            'MantisCore' => '1.2.0, &lt;= 1.2.0',  #   Should always depend on an appropriate version of MantisBT
-            );
+        $this->requires = array(    # Plugin dependencies
+            'MantisCore' => '1.3, &lt; 3.0',  # Should always depend on an appropriate
```

Comment:
I intentionally used `1.3, < 3.0` range to illustrate the case of a plugin spanning 2 major Mantis versions. 

I guess it makes sense to use a more common use case for the example. Will change that.